webcheck/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.9'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: webcheck_server_1
+      APP_PORT: 3000
+      PROXY_AUTH_ADD: "false"
```

Comment:
Shall we remove `PROXY_AUTH_ADD: "false"` entirely so that this app gets umbrel's transparent auth proxy? 

That way the WebCheck UI inherits the security properties of our umbrel auth:

- By default, we only let requests through the proxy if the user already has a valid auth cookie from the Umbrel homescreen. So since the WebCheck dashboard has no auth of its own, it will still be protected by ours. This gives a good UX because there's zero friction for Umbrel users... they can just access the dashboard without re-entering any credentials if they are already logged in to their Umbrel.

- It also has the benefit of inheriting other security properties of Umbrel auth, such as 2FA if they have it enabled on their Umbrel. They would then get 2FA security for all apps behind the auth proxy too.

What do you think?